hardhat.config.ts

```diff
@@ -3,6 +3,9 @@ import '@nomicfoundation/hardhat-toolbox';
 
 const config: HardhatUserConfig = {
 	solidity: '0.8.17',
+	mocha: {
+		timeout: 30000,
```

Comment:
While testing addMailRecipients with 220 recipients timeout can occur - therefore such a high timeout limit to prevent falsely failed tests.